### PR TITLE
Pass headers to tileLayer

### PR DESCRIPTION
**Pull request by dpramar**

### CODE REVIEW

**Review by Sumolari — APPROVED** [review]
Are we sure we want to include the minified file in the Git repo?

File: dist/angular-leaflet-directive.no-header.js [hunk 1]
```diff
@@ -2061,11 +2061,12 @@ angular.module('leaflet-directive')
       apiKey: layerDefinition.apiKey,
       pluginOptions: layerDefinition.pluginOptions,
       user: layerDefinition.user,
+      headers: layerDefinition.headers,
     };
 
     //TODO Add $watch to the layer properties
     if (layerTypes[layerDefinition.type]) return layerTypes[layerDefinition.type].createLayer(params);
-    return L.tileLayer[layerDefinition.type](params.url, params.options);
+    return L.tileLayer[layerDefinition.type](params.url, params.options, params.headers);
```
Comment (fofi) [inline dist/angular-leaflet-directive.no-header.js:2069]: The name of this file is angular-leaflet-directive.**no-header**.js, is there already one with header?

**Review by fofi — APPROVED** [review]

File: dist/angular-leaflet-directive.no-header.js (hunk 1 above)
Comment (dpramar) [reply dist/angular-leaflet-directive.no-header.js:2069]: This was before, the name of the file is not related with my change of adding the headers. I don't know the purpose of the file.
All `dist/` is created automatically when running `build` task

Comment (Sumolari) [reply dist/angular-leaflet-directive.no-header.js:2069]: Do we really want to keep `dist` folder in the git repo? Isn't `dist` folder pushed to NPM?

Comment (dpramar) [reply dist/angular-leaflet-directive.no-header.js:2069]: Currently in our package.json we pick the commit, so I don't think that npm is used.
About the `dist`, it looks like that is how the main `angular-leaflet-directive` builds, we haven't change anything of that, only small tweaks for our use case.

If we want to change that, probably should be tackle in another task, if worth the time.

Comment (Sumolari) [reply dist/angular-leaflet-directive.no-header.js:2069]: Didn't remember we were using git to install this package

Create a task to move to NPM in the future if you don't mind

**Pull request merged**